pkg/scaffold/.eslintrc.js

```diff
@@ -149,6 +149,9 @@ module.exports = {
     // we're using empty functions as fallback for undefined props
     "@typescript-eslint/no-empty-function": "off",
 
+    // For pouchdb and couchdb snake case conventions
+    "@typescript-eslint/camelcase": "off",
+
```

Comment:
Can we do this on a per-line/per-file basis, rather than in all of the packages in the entire monorepo?